pages/index.tsx

```diff
@@ -0,0 +1,79 @@
+import Image from 'next/image';
+import type { NextPage } from 'next';
+
+import { Grid, GridItem, H1, Stack } from '@smg-automotive/components';
+
+import TableOfContents from 'components/home/TableOfContents';
+import Requirements from 'components/home/Requirements';
+import Glossary from 'components/home/Glossary';
+import Figure from 'components/home/Figure';
+import Disclaimer from 'components/home/Disclaimer';
+import Description from 'components/home/Description';
+import ApiDocs from 'components/home/apiDocs';
```

Comment:
this was probably a lot of work, but I find the embedded api docs quite overkill. a link to the public swagger would be enough for me. I _do like_ having everything in one place, but it can generate unnecessary maintenance effort and opens the possibility for human errors causing hiccups/misunderstandings when solving the challenge

up to you :)

Reply:
There currently is no public swagger, that's the only reason I created those.

Reply:
argh, thanks for the extra effort!